applications/rag/example_notebooks/rag-kaggle-ray-sql-interactive.ipynb

```diff
@@ -0,0 +1,425 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "304bba57-cdb4-42d5-a4b7-2494b6cfa4ff",
+   "metadata": {},
+   "source": [
+    "# RAG-on-GKE Application\n",
+    "\n",
+    "This is a Python notebook for generating the vector embeddings used by the RAG on GKE application. For full information, please checkout the GitHub documentation [here](https://github.com/GoogleCloudPlatform/ai-on-gke/blob/main/applications/rag/README.md).\n",
+    "\n",
+    "\n",
+    "## Setup Kaggle Credentials\n",
+    "\n",
+    "First we will setup your Kaggle credentials. Replace the following with your own settings from the Kaggle web page. Navigate to https://www.kaggle.com/settings/account and generate an API token to be used to setup the env variable. See https://www.kaggle.com/docs/api#authentication how to create one."
```

Comment:
nit: "to be used to setup the env variable" is kind of weird. Maybe just have the instructions for how to create Kaggle creds and ask users to replace the specific variables (KAGGLE_USERNAME and KAGGLE_KEY) in the cell below with their Kaggle creds?

Reply:
Merged the cells.